torch/csrc/autograd/generated/python_torch_functions.cpp

```diff
@@ -2606,7 +2607,7 @@ static PyObject * THPVariable_bartlett_window(PyObject* self_, PyObject* args, P
         .layout(r.layout(2).layout)
```

Comment:
We can delete this TensorOptions now, right?

Reply:
Yep